### DISCUSSION TITLE
New GitHub Feature - Remembering Deceased Team Members

### DISCUSSION THREAD

**Discussion by WebCrew** [community/community#18435] [Other Feature Feedback, Questions, & Ideas]
**Hallo GitHub Team & Community**,

when I found out a few months ago that a young developer who helped me with many of my repositories unfortunately passed away at the age of only 23 I was very very sad and I was looking for a way to honor her on my profile, but there was none.

I would like to have a small place (perhaps right under the profile description) where one can mourn honored team members and friends who have passed away and leave a memorial for them.

What do You think about it?

Kind regards,

Andy ;)

**Comment by queenofcorgis** [comment]
@WebCrew I am so sorry to hear about your loss ❤️ . Thank you for your feedback, I've added the proper label on this. If you have any ideas or examples of this, please do share them!

**Reply by WebCrew** [reply]
Thanks Lili for the kind words and for the right lable.

![thx](https://user-images.githubusercontent.com/12986067/173729802-7cf220c2-dfd1-4198-bab0-f97cc83153a1.gif)

I wish You a great day.

Regards, Andy ;)

**Comment by Ngquang** [comment]
yep. a great way to remember them.

**Reply by WebCrew** [reply]
Thanks for Your opinion :)

Kind regards, Andy ;)
